content/servicemesh/visualize.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Monitor & Visualize"
+date: 2018-11-13T22:42:01+09:00
+weight: 60
+draft: false
+---
+
+### Collecting new telemetry data
+
+Download a YAML file to hold configuration for the new metric and log stream that Istio will generate and collect automatically.
+
+```
+curl -LO https://s3.ap-northeast-2.amazonaws.com/docshared/AWS/istio-telemetry.yaml
```

Comment:
Can we move this file to the workshop repo? check out https://eksworkshop.com/logging/deploy/ for an example of including static files.

Reply:
moved the file to the workshop repo.